include/frg/formatting.hpp

```diff
@@ -677,6 +684,24 @@ namespace detail_ {
 			}
 		}
 	};
+
+	#ifdef FRIGG_HAS_IOSTREAM
```

Comment:
preprocessor macros go on column zero and can get indentation after the #, in their own "namespace"